docs/api/api_docs.yaml

```
@@ -2027,91 +2043,220 @@ paths:
     get:
       responses:
         '200':
-          description: OK
+          description: Retrieved replication configurations successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/All-replications'
+        '404':
+          $ref: '#/components/responses/Not-found'
       tags:
         - Admin
+      summary: Get all replication configurations
+      description: |-
+        This will retrieve all the defined replication configs as a key map.
+
+        The `assigned_node` fields will end with `(local)` or `(non-local)` depending on if the replication is local or not.
```

Defining local with local doesn't clarify much 😄  but also shouldn't this really be in the response schema description instead of the endpoint description?

```suggestion
        The `assigned_node` fields will end with `(local)` or `(non-local)` depending on if the replication is running on this sync gateway node.
```